Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/PurposeOfForDecider.java

```diff
@@ -90,7 +86,12 @@ public boolean isPurposeFor(CallbackProperties properties) {
         }
 
         if (LOG.isDebugEnabled()) {
-            logPurposeDecision(purposeFor, hcid, serviceName.getUDDIServiceName());
+            if (serviceName != null){
```

Comment:
this if else block can be re-written as  logPurposeDecision(purposeFor, hcid, serviceName.getUDDIServiceName()!=null? serviceName.getUDDIServiceName():null)